docs/application/web/guides/applications/addon.md

```diff
@@ -0,0 +1,202 @@
+        "version”: “0.0.1”,
+        “main”: “addonSampleAd”,
+        “settings”: “”,
+        “description”: “Sample to show how to show up advertisement at prelaunch event”
+    }
+    ```
+
+### Main JavaScript File (mandatory)
+
+This should have the same name with ‘main’ in manifest
+
+This will include :
+
+  - `activate()` : Registering a callback to an addon event should happen here
+  - `deactivate()` : Unregistering the callback to the addon event should happened here
+  - `Callback Functions` : Addon developers write their own code to define the behavior which happens when addon events occur
+
+Here’s an example of an main JavaScript file
+
+    ```
+    class AddonSampleAd {
+        constructor(em) {
+          console.log('AddonSampleAd constructor');
+          this.evt = em;
+        }
+        activate() {
+            console.log('AddonSampleAd.activate');
+            this.evt.on('lcPrelaunch', this.prelaunch);
+        }
+        deactivate() {
+            console.log('AddonSampleAd.dectivate');
+            this.evt.off('lcPrelaunch', this.prelaunch);
+        }
+        prelaunch(winId, orgUrl) {
+            console.log('AddonSampleAd.prelaunch');
+        }
+    }
+    module.exports = AddonSampleAd;
+
+    ```
+
+<a name="event"></a>
+## Event
+
+| Event                                      | Description                              |
+| ------------------------------------------ | ---------------------------------------- |
+| lcPrelaunch | Emitted once for web app launching just before the main page is shown |
+| lcResume | Emitted when the app is resumed |
+| lcSuspend | Emitted when the app is suspended |
+| lcQuit | Emitted before the app is ended |
+| contentDidFinishLoad | Emitted when the page navigation is done | 
+
+<a name="mechanism"></a>
+## Mechanism
+
+Addons cannot run in standalone. They are run when any web application runs. They work on event-driven invocation. Each addon registers its callbacks to events which are pre-defined in WRTjs. Registered callbacks will be invoked whenever the registered events occur during web application’s life cycle.  
+
+**Figure: Addon Mechanism**
+
+![Addon mechanism](./media/addon_mechanism.png)
+
+<a name="compare"></a>
+## Comparing to Chrome Extensions
+
+In the way to reach here, you might think that there’s no difference from Chrome Extensions.
+
+1. Applying to Multiple Applications
+
+   Chrome Extensions are applied to multiple pages (or tabs) in an application, Chrome. However, addons are applied to multiple web applications in a platform, Tizen at this moment.
+
+2. Wide Range of Customization
+
+   Addons can customize with related to each web application’s life cycle events such as Prelaunch, Resume, Suspend or Quit. Addons get the notification once they register its own callback for a lifecycle event. That’s possible because they are run on WRTjs, also known as Web Runtime JS, which manages and controls from birth to death of web applications.
```

Comment:
Done.

Addons can give change on the action according to the Web application's life cycle events.
Therefore, I'll rephrasing to as below:

Addons can customize an action when each Web application’s life cycle event such as Prelaunch, Resume, Suspend, or Quit occurs. Addons get the notification after they register its own callback for a lifecycle event. That is possible because they run on WRTjs, also known as Web Runtime JS, which manages and controls from birth to death of Web applications.